util/config/schema.py

```diff
@@ -1609,15 +1609,36 @@
                     "description": "name of service in otel spans",
                     "x-example": "quay",
                 },
-                "dt_api_url": {
+                "OTEL_EXPORTER_OTLP_ENDPOINT": {
                     "type": "string",
-                    "description": "url for dynatrace api",
-                    "x-example": "https://dynatrace-api.example",
+                    "description": "url for OTLP api environment overwrites",
+                    "x-example": "https://dynatrace-api.example:443",
                 },
-                "dt_api_token": {
+                "OTEL_EXPORTER_OTLP_TRACES_ENDPOINT": {
                     "type": "string",
-                    "description": "token for dynatrace api",
-                    "x-example": "sometoken",
+                    "description": "url for OTLP api environment overwrites",
+                    "x-example": "https://dynatrace-api.example:443",
+                },
+                "OTEL_EXPORTER_OTLP_HEADERS": {
```

Comment:
Can you still configure it without a collector or sidecar? Could we still put the previous `dt_api_url` and `dt_api_token` in `OTEL_EXPORTER_OTLP_ENDPOINT` and `OTEL_EXPORTER_OTLP_HEADERS` and have backward compatible behavior?